bundle/tests/scorecard/kuttl/deployment-strategy/01-update-application.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: app.stacks/v1beta1
```

Comment:
> For the startup probe test, is it worth modifying step 1 of the test to change the times of the startup probe? that would match what the liveliness/readiness sections of the test do

Yes, I will add modification to startup probe times in test 1.

> Is this whole step necessary? The only change is to the application image, and the rollout itself is handled by kube, so it feels a bit as if we are testing the kube rollout strategy. Unless I've missed something, I think it would probably be better to keep the test short and remove this step
But it might be worth adding a step to explicitly set the strategy to recreate.

I will remove this step and keep test 3 for specifying strategy to Recreate or OnDelete.